include/wlr/backend/drm.h

```diff
@@ -13,6 +13,7 @@
 #include <wlr/backend.h>
 #include <wlr/backend/session.h>
 #include <wlr/types/wlr_output.h>
+#include <xf86drmMode.h>
```

Comment:
Maybe it would be better to forward declare `typedef struct _drmModeModeInfo drmModeModeInfo;` to allow users to use this file without having to link with libdrm?

Reply:
Hmm, makes sense